src/binder/expression/agg_call.rs

```diff
@@ -89,10 +89,7 @@ impl Binder {
             }
         }
         let (kind, return_type) = match func.name.to_string().to_lowercase().as_str() {
-            "avg" => (
-                AggKind::Avg,
-                Some(DataType::new(DataTypeKind::Double, false)),
-            ),
```

Comment:
I'm a little bit unsure and may need some time to look into the details of how the type system work. 🤔

Reply:
Avg will be rewritten, so I think it's okay to fill any type here.